client/src/app/slides/motions/motion/motions-motion-slide.component.ts

```diff
@@ -1,14 +1,258 @@
 import { Component } from '@angular/core';
 import { BaseSlideComponent } from 'app/slides/base-slide-component';
 import { MotionsMotionSlideData } from './motions-motion-slide-data';
+import { ChangeRecoMode, LineNumberingMode } from '../../../site/motions/models/view-motion';
+import { DomSanitizer, SafeHtml } from '@angular/platform-browser';
+import { ConfigService } from '../../../core/ui-services/config.service';
+import { DiffService, LineRange } from '../../../core/ui-services/diff.service';
+import { LinenumberingService } from '../../../core/ui-services/linenumbering.service';
+import { ViewUnifiedChange } from '../../../shared/models/motions/view-unified-change';
+import { MotionsMotionSlideObjChangeReco } from './motions-motion-slide-obj-change-reco';
 
 @Component({
     selector: 'os-motions-motion-slide',
     templateUrl: './motions-motion-slide.component.html',
     styleUrls: ['./motions-motion-slide.component.scss']
 })
 export class MotionsMotionSlideComponent extends BaseSlideComponent<MotionsMotionSlideData> {
-    public constructor() {
+    /**
+     * Indicates the LineNumberingMode Mode.
+     * @TODO Read value from the backend
+     */
+    public lnMode: LineNumberingMode = LineNumberingMode.Outside;
```

Comment:
Please add these values to the `projector.py` and read them there from the config.